subt/main.py

```diff
@@ -862,15 +814,9 @@ def test_nav_trace(self):
         dy -= 5.000014
         trace = Trace()
         trace.add_line_to((3 - dx, -5 - dy, 0))  # before tunnel
```

Comment:
I don't really understand what these lines do. Why do we have
```
dx -= -7.500005
dy -= 5.000014
```
Those are really strange numbers. The `dx` and `dy` should be coordinates of the middle of the entrance, right? Or I am confused.

Reply:
well, I tried to minimize the changes and these crazy numbers are from time to switch to absolute coordinates for all robots. It can be nicer now ... yes and the name `test_` is also misleading, you are right.

Reply:
ok, I understand, but please don't try to minimize changes - instead, if at all possible - try to maximize readability - like "what would be the most obvious way to code this behavior if I were to do it today from scratch?" - I would very much appreciate that, thanks